### PR TITLE
Add task solution

### PR DESCRIPTION
**Pull request by SerhiiUnhurian**
[DEMO LINK](https://SerhiiUnhurian.github.io/react_tabs-js/)

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+iframe {
+  display: none;
+}/*# sourceMappingURL=App.css.map */
```
Comment (anastasiiavorobiova) [inline src/App.css:3]: Consider fixing the error here 
Screenshot 2023-11-05 at 19 10 37

File: src/App.jsx [hunk 2]
```diff
@@ -1,40 +1,40 @@
...
-            <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-          </li>
-        </ul>
-      </div>
-
-      <div className="block" data-cy="TabContent">
-        Some text 1
+        <div className="block" data-cy="TabContent">
+          {currentSelectedTab.content}
+        </div>
```
Comment (anastasiiavorobiova) [inline src/App.jsx:36]:
```suggestion

```
Move this block to the tab component

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done! Make sure that your link to the deployed page works

File: src/App.jsx (hunk 2 above)
Comment (SerhiiUnhurian) [reply src/App.jsx:36]: but where?
in that component only tabs, is i add it there if will appear right besides them

File: src/components/Tabs/Tabs.jsx [hunk 3]
```diff
@@ -1 +1,26 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <>
+      {tabs.map(tab => (
+        <li
+          className={selectedTab.id === tab.id ? 'is-active' : ''}
```
Comment (etojeDenys) [inline src/components/Tabs/Tabs.jsx:8]: we usually use classnames package for such cases

File: src/App.jsx [hunk 4]
```diff
@@ -1,40 +1,36 @@
...
-        Some text 1
+      <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+          <ul>
+            <Tabs
+              tabs={tabs}
+              selectedTabId={selectedTabId}
+              onTabSelected={tab => setSelectedTabId(tab.id)}
+            />
+          </ul>
```
Comment (etojeDenys) [inline src/App.jsx:31]: better to move this `ul` into Tabs if you use `li`s there, it will improve the reusability

**Review by etojeDenys — APPROVED** [review]
good job

File: src/App.jsx (hunk 4 above)
Comment (SerhiiUnhurian) [reply src/App.jsx:31]: okay)

File: src/components/Tabs/Tabs.jsx (hunk 3 above)
Comment (SerhiiUnhurian) [reply src/components/Tabs/Tabs.jsx:8]: its just more convient for me to use ternari here, if we use it just once

File: src/App.jsx (hunk 4 above)
Comment (SerhiiUnhurian) [reply src/App.jsx:31]: but now text displays not under tabs, but next to then

**Pull request closed without merging**